hal/i2c_api.h

```diff
@@ -38,47 +38,47 @@
  *
  * The I2C hal provides a low level interface to the I2C interface of a target.
  *
- * # Defined behaviour
- * * The function ::i2c_init initialises the peripheral pins specified in the input parameters,
- * initialises the peripheral in master mode if `is_slave` is false,
- * initialises the peripheral in slave mode if `is_slave` is true and `supports_slave_mode` is true  - Verified by test ::fpga_i2c_test_init_free.
- * * The function ::i2c_free resets the pins used to initialise the peripheral to their default state,
+ * # Defined behavior
+ * * The function ::i2c_init initializes the peripheral pins specified in the input parameters,
+ * initializes the peripheral in master mode if `is_slave` is false,
+ * initializes the peripheral in slave mode if `is_slave` is true and `supports_slave_mode` is true  - Verified by test ::fpga_i2c_test_init_free.
+ * * The function ::i2c_free resets the pins used to initialize the peripheral to their default state and
  * disables the peripheral clock - Verified by test ::fpga_i2c_test_init_free.
  * * The function ::i2c_get_capabilities fills the contents of the `i2c_capabilities_t` parameter - Verified by test ::fpga_i2c_test_get_capabilities.
- * * The function ::i2c_frequency sets the frequency to use for the transfer, returns the actual frequency that will be used,
+ * * The function ::i2c_frequency sets the frequency to use for the transfer, returns the actual frequency used and
  * must leave all other configuration unchanged - Verified by test ::fpga_i2c_test_frequency.
```

Comment:
Missing `i2c_set_clock_stretching` description should go after `i2c_frequency`:

`The function ::i2c_set_clock_stretching enables or disables clock stretching for the peripheral when in slave mode, does nothing if called in master mode - not tested.`